.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.3.0
```

Comment:
I guess I didn't need to commit this. rvenv doesn't recognize 2.3.1 yet and I wasn't willing to fight with it.